DESCRIPTION

```diff
@@ -9,22 +9,35 @@ Authors@R: c(
            email = "p.dutey-magni@ucl.ac.uk",
            comment = c(ORCID = "0000-0002-8942-9836")),
     person("University College London", role = c("cph")))
-Description: >
-   A package to manipulate the SNOMED clinical ontology using the 
-   SNOMED CT Terminology Server REST API <https://github.com/IHTSDO/snowstorm>.
+Description: Manipulate the SNOMED CT clinical ontology using the 
```

Comment:
Sorry this is tiny really, but not having seen the code but I wonder whether it is really "manipulate SNOMED-CT", which I understand to mean change the ontology as such